Android/ARCore/source/Properties/AssemblyInfo.cs

```diff
@@ -18,7 +18,7 @@
 // The form "{Major}.{Minor}.*" will automatically update the build and revision,
 // and "{Major}.{Minor}.{Build}.*" will update just the revision.
 
-[assembly: AssemblyVersion("1.0.0")]
+[assembly: AssemblyVersion("1.7.0")]
```

Comment:
We should probably keep this as 1.0 and then add the [AssemblyFileVersion]. This [AssemblyVersion] represents the actual assembly version and will require the app to be recompiled.